### PR TITLE
[14.0][ADD] rma_recall

### PR DESCRIPTION
**Pull request by Vimal-OSI**

### CODE REVIEW

**Comment by JordiBForgeFlow** [comment]
@MateuGForgeFlow Can you review this one?

File: rma_recall/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    "name": "RMA Recall",
+    "version": "14.0.1.0.0",
+    "license": "LGPL-3",
+    "category": "RMA",
+    "summary": "To track inventory items by lot number",
+    "author": "Open Source Integrators, Odoo Community Association (OCA)",
+    "website": "https://github.com/OCA/https://github.com/ForgeFlow/stock-rma",
```
Comment (MateuGForgeFlow) [inline rma_recall/__manifest__.py]: Mixed Websites

File: rma_recall/models/rma_recall.py [hunk 2]
```diff
@@ -0,0 +1,195 @@
...
+                "New"
+            )
+        return super().create(vals)
+
+    def _valid_field_parameter(self, field, name):
+        # I can't even
+        return name == "tracking" or super()._valid_field_parameter(field, name)
+
+    def action_search(self):
+        MoveLine = self.env["stock.move.line"]
```
Comment (MateuGForgeFlow) [inline rma_recall/models/rma_recall.py]: Consider using lower-case for variables!

File: rma_recall/views/rma_recall_view.xml [hunk 3]
```diff
@@ -0,0 +1,183 @@
...
+        <field name="res_model">rma.recall</field>
+        <field name="view_mode">tree,form</field>
+    </record>
+
+    <!-- Menu items -->
+    <menuitem
+        id="rma_recall_menu"
+        name="Recalls"
+        parent="rma.menu_rma_root"
+        sequence="20"
```
Comment (MateuGForgeFlow) [inline rma_recall/views/rma_recall_view.xml:179]: Maybe change the sequence in order to have the Recalls menu after the customer and supplier RMA menus.

File: rma_recall/__manifest__.py [hunk 4]
```diff
@@ -0,0 +1,19 @@
+# Copyright (C) 2021 Open Source Integrators (https://www.opensourceintegrators.com)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    "name": "RMA Recall",
+    "version": "14.0.1.0.0",
+    "license": "LGPL-3",
```
Comment (MateuGForgeFlow) [inline rma_recall/__manifest__.py:7]: Mixed licenses in files and manifest

**Review by MateuGForgeFlow — COMMENTED** [review]
Hi @Vimal-OSI, some comments for the module! Thank you!

File: rma_recall/views/rma_recall_view.xml [hunk 5]
```diff
@@ -0,0 +1,183 @@
...
+                                name="rma_date"
+                                attrs="{'readonly': [('state', '!=', 'new')]}"
+                            />
+                            <field
+                                name="origin"
+                                attrs="{'readonly': [('state', '!=', 'new')]}"
+                            />
+                        </group>
+                    </group>
+                    <group cols="4">
```
Comment (RLeeGML) [inline rma_recall/views/rma_recall_view.xml]:
```suggestion
                    <group>
```
The display is messed up with a fixed col width @Vimal-OSI

**Comment by Vimal-OSI** [comment]
@MateuGForgeFlow @RLeeOSI , Requested changes are done, can you please re-review it? Thanks

**Comment by MateuGForgeFlow** [comment]
Hi @Vimal-OSI, functional review LGTM!
It would be good to add some tests to the module! Thank you!

**Pull request closed without merging**